lib/output.js

```diff
@@ -865,15 +865,21 @@ function OutputStream(options) {
     });
 
     PARENS(AST_Number, function(output){
+        if (!output.option("galio")) return false;
+        // https://github.com/mishoo/UglifyJS/pull/1009
+
         var p = output.parent();
-        if (p instanceof AST_PropAccess && p.expression === this) {
-            var value = this.getValue();
-            if (value < 0 || /^0/.test(make_num(value))) {
-                return true;
-            }
-        }
+        return p instanceof AST_PropAccess && p.expression === this && /^0/.test(make_num(this.value));
     });
 
+    // PARENS(AST_BigInt, function(output){
+    //     if (!output.option("galio")) return false;
+    //     // https://github.com/mishoo/UglifyJS/pull/1009
+
+    //     var p = output.parent();
+    //     return p instanceof AST_PropAccess && p.expression === this && /^0/.test(make_num(this.value));
+    // });
```

Comment:
not sure if we need this, but given it's for "galio", prob not important initially 🤔